Game.java

```diff
@@ -34,18 +35,14 @@ public String showBoard(Board board) {
     }
 
     public Board playerMakesMove() {
-        int index = randInt(0, board.availablePositions().size());
-        int pos;
-        if (turn == 'o' && gameMode.equals("pvc")) {
-            pos = (int) board.availablePositions().get(index);
-            board = board.playerMakesMove('o', pos);
+        if (getTurn() == 'o' && gameMode.equals("pvc")) {
```

Comment:
why does getTurn()  == 'o' matter here? 
I it isn't 'o' and gameMode is pvc then the default 'else will be entered. 
This could be tightened up as difficult to understand the flow.